Directory.Build.props

```diff
@@ -2,8 +2,8 @@
 
   <PropertyGroup>
     <VersionPrefix>1.3.0</VersionPrefix>
```

Comment:
I believe this PR is a breaking change no? Not opposed! Just double checking

Reply:
I *mostly* put this PR up to get the thing building locally *at all*. Marked this "Draft" to reflect that.